ci/matrix.xml

```diff
@@ -65,13 +65,13 @@
 
   <task id="test-suite-ecma-script5"><![CDATA[
     setJavaVersion $java
-    sbtretry ++$scala testSuite/test &&
+    sbtretry ++$scala $testSuite/test &&
     sbtretry 'set scalaJSUseRhino in Global := false' \
-        ++$scala testSuite/test &&
+        ++$scala $testSuite/test &&
     sbtretry 'set scalaJSUseRhino in Global := false' \
-	'set scalaJSStage in Global := FullOptStage' \
-        ++$scala testSuite/test \
-        testSuite/clean &&
+	    'set scalaJSStage in Global := FullOptStage' \
```

Comment:
There was a thab there hidden at the beginning of that line, I'll clean up all the tabs in this file.